algorithms/refinement/reflection_manager.py

```diff
@@ -314,14 +319,23 @@ def from_parameters_reflections_experiments(
                 *params.weighting_strategy.constants, stills=do_stills
             )
 
+        # Check for deprecated parameter
+        if params.trim_scan_edges > 0.0:
+            warnings.simplefilter("default")
+            warnings.warn(
+                "The parameter trim_scan_edges is deprecated and will be removed shortly",
+                DeprecationWarning,
```

Comment:
```suggestion
                DeprecationWarning,
                stacklevel=2,
```
I find warnings much more helpful when they indicate the line where a function was called from rather than the line where the warning itself is created.